pkg/crc/machine/start.go

```diff
@@ -227,17 +227,16 @@ func configureSharedDirs(vm *virtualMachine, sshRunner *crcssh.Runner) error {
 	}
 	logging.Infof("Configuring shared directories")
 	for _, mount := range sharedDirs {
-		// CoreOS makes / immutable, we need to handle this if we need to create a directory outside of /home and /mnt
-		isHomeOrMnt := strings.HasPrefix(mount.Target, "/home") || strings.HasPrefix(mount.Target, "/mnt")
-		if !isHomeOrMnt {
+		// Try to create the mount directory and if it fails then
+		// make the file system mutable and again try to create the
+		// mount directory
```

Comment:
I'd mention explicitly that if the directory already exists, then `mkdir -p` won't return an error.